### PR TITLE
#40 - Added global tests timeout

### PR DESCRIPTION
**Pull request by FarmerSez**
Solves #40

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -12,6 +12,10 @@ commands =
 	pydocstyle -v src/
 	py.test --cache-clear -vv src/
 
+[pytest]
+timeout = 10
+timeout-method = thread
```
Comment (snudler6) [inline tox.ini]: If you are not patching purest, why threading is needed?

File: src/time_travel/patchers/base_patcher.py [hunk 2]
```diff
@@ -6,7 +6,9 @@
 
 class BasePatcher(object):
     """Base class for patching time and I/O modules."""
-    
+
+    UNPATCHED_MODULES = ['pytest', '_pytest', 'unittest', 'mock', 'threading']
```
Comment (snudler6) [inline src/time_travel/patchers/base_patcher.py]: Document?

File: tox.ini (hunk 1 above)
Comment (FarmerSez) [reply tox.ini]: That is, infact, correct.

File: src/time_travel/patchers/base_patcher.py (hunk 2 above)
Comment (FarmerSez) [reply src/time_travel/patchers/base_patcher.py]: Tell me if you want more.

Comment (snudler6) [reply src/time_travel/patchers/base_patcher.py]: Cool

**Pull request merged**